packages/esm-patient-registration-app/src/patient-registration/input/custom-input/autosuggest/autosuggest.component.tsx

```diff
@@ -150,10 +160,11 @@ export const Autosuggest: React.FC<AutosuggestProps> = ({
   return (
     <div className={styles.autocomplete} ref={wrapper}>
       <label className="cds--label">{labelText}</label>
-      <Layer>
+      <Layer className={invalid ? classNames(styles.invalid) : {}}>
```

Comment:
Uh... the idea here was that you can do:


```suggestion
      <Layer className={classNames({[styles.invalid]: invalid})}>
```

Reply:
Then there's no weirdness of accidentally adding a class called `[object Object]`.